heareval/tasks/dcase2016_task2.py

```diff
@@ -9,17 +9,21 @@
 """
 
 import logging
-import os
 from pathlib import Path
 from typing import List
 
 import luigi
 import pandas as pd
 
 import heareval.tasks.pipeline as pipeline
+import heareval.tasks.util.luigi as luigi_utils
 
 logger = logging.getLogger("luigi-interface")
 
+# This percentage should not be change as this decides
```

Comment:
I mean shouldn't this just be global and not task specific? If there is not validation, use 20% as validation. If there is not test, use 20% as test. I don't see why it should be task specific

Reply:
I have moved it into the pipeline and made a generic split function that will handle all splits scenarios with the global percentage for test and validation. This variable can be moved out of the `pipeline.py` in a better place, but logically is a global variable now.